src/pages/design/controls.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import Layout from '../../components/Layout'
+import GuidelineHeader from '../../components/guideline-header/guideline-header'
+import GuidelineLeftNav from '../../components/guideline-left-nav/guideline-left-nav';
+import './controls.scss'
+import { graphql, Link, StaticQuery } from "gatsby";
+import Img from 'gatsby-image';
+
+class DesignControlsIndexPage extends React.Component {
```

Comment:
I think longer term we should make this into a component.
Taking out the following and moving to `design/guideline.js`
```
<Layout>
        <GuidelineHeader></GuidelineHeader>
        <GuidelineLeftNav></GuidelineLeftNav>
```